### DISCUSSION TITLE
How do I set Dewalt Battery under-voltage protection

### DISCUSSION THREAD

**Discussion by doubaz222** [Ralim/IronOS#1065] [Q&A] [answered]
Hi guys, I am very new to this so please don't crucify me :)

So I am trying to power my ts100 with a dewalt 18v battery and wanted to set some protection for the battery with the firmware, however when I choose DC power source in the power settings, it won't let me set an under voltage protection and the only other options are for LiPo batteries. 

Is there a way to set a custom under-voltage protection, I probably need about 14v but i'm only guessing...

I guess that it might be worth noting the the batteries themselves don't have undercurrent protection themselves unlike many other brands because the tools usually handle that side of things.

Thanks!

**Comment by CRImier** [comment]
Would guess the Dewalt batteries are 5s (3.6V*5 = 18V), so you should be fine if you pick that option in the menu.

**Comment by Ralim** [comment, marked as answer]
So the DC option is short for "no protections as im not using a battery"

18V Drill batteries are 5s battery packs. So set it to 5S and then you can adjust the cutoff voltage per cell to select where it turns off at.
3-3.3V are safe options, down to 2.8V or so is generally in spec for most cells.

**Reply by doubaz222** [reply]
Wow thanks so much guys for your quick replies, so to follow up on this, If I leave it permanently in 5s with 3.3v to be conservative. I can still use my mains power brick ot the batteries. (mains power brick is 24v 3A)

**Reply by Ralim** [reply]
Yep, it only sets a lower limit, so since that is less than your power brick it will work as you want

**Reply by doubaz222** [reply]
You're amazing thanks again!
